src/app/public/modules/repeater/repeater-item.component.html

```diff
@@ -1,10 +1,19 @@
 <div
   class="sky-repeater-item sky-padding-even-default"
+  [attr.tabindex]="tabIndex"
   [ngClass]="{
     'sky-repeater-item-active': isActive,
     'sky-repeater-item-collapsible': isCollapsible,
     'sky-repeater-item-selected': isSelected
   }"
+  (click)="onRepeaterItemClick($event)"
+  (keydown.ArrowDown)="onArrowDown($event)"
+  (keydown.ArrowLeft)="onArrowLeft($event)"
+  (keydown.ArrowRight)="onArrowRight($event)"
+  (keydown.ArrowUp)="onArrowUp($event)"
+  (keydown.Enter)="onEnter($event)"
+  (keydown.Space)="onSpace($event)"
```

Comment:
This will add six event listeners to every repeater item on the page. It would be better to have only one keydown event and then capture the `event.key` in a switch statement.

Reply:
Why is that better? Is your concern performance or legibility?

Reply:
Performance. Having only one event listener versus six will optimize performance, especially since we don't know how many repeater items will be added to the page.

Reply:
Would you be willing to try and put all of these event listeners into a single `keydown` listener? I've looked through your handler methods and there's a lot of repeated code; I think the readability and reusability of your code would improve if we combined them into one.

Reply:
Done.